### ISSUE TITLE
Document reason for removing specific issues from A11y statement

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#2595]
We need to itemize issues that we're removing from the statement a reference to the card/task that resolves the issue.
This will give us an easy reference if we need it in future.

Issues to remove will be entered here once identifiied and can be checked off once card/task resolving the issue is indicated in 
Details about reason for removal in Phil's [Audit Fixes Tracking](https://docs.google.com/spreadsheets/d/1V2gqBBUkODQFoGfqSDUQe5cpMe5xBWCOKFLwzhBkiwc/edit?gid=0#gid=0)

- [x] Only one way to navigate to different pages; To find a different page, the only option is using GC Notify’s headers
- [x] Sometimes GC Notify does not give you enough information to prevent errors GC Notify does not always: Show a difference between errors and warnings.
- [x] Sometimes GC Notify does not give you enough information to prevent errors GC Notify does not always: indicate errors using the correct coding language.
- [x] On the Templates page, button names change without warning
Under templates, there are buttons for “Create template”, “Copy template” and “New folder”. If you select a template before selecting a button, the buttons change to “Move” and “Add new folder”.
- [x] Safari does not clearly communicate pop-up menus.
- [x] When checking the number of message failures, you cannot distinguish between email and text messages.

**Comment by YedidaZalik** [comment]
documentation is complete
